composer.json

```diff
@@ -29,6 +29,7 @@
         "opis/closure": "^3.1",
         "psr/container": "^1.0",
         "psr/simple-cache": "^1.0",
+        "psr/http-message": "^1.0",
```

Comment:
Please add this as a suggested dependency, and also update the `src/Illuminate/Filesystem/composer.json` file.

Reply:
Thanks for the feedback. I've changed it to a suggested dependency in the root composer.json and added it as a suggested dependency in the filesystem composer.json